app/controllers/api/service_requests_controller.rb

```diff
@@ -35,5 +35,16 @@ def service_requests_search_conditions
       return {} if @auth_user_obj.admin?
       {:requester => @auth_user_obj}
     end
+
+    def add_approver_resource(type, id, data)
+      raise 'Must provide a user id' unless data['user_id']
+      user = User.find(data['user_id'])
```

Comment:
let's support for user_id as well as the API user reference hash (via href or id via parse_id) for compatibility.  Thanks !!